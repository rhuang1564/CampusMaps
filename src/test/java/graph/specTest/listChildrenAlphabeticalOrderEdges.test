# Tests alphabetical ordering of listening of a node with multiple edges between same node

# Create a graph
CreateGraph A

# Add a node
AddNode A node1
AddNode A node2

# Add edges
AddEdge A node1 node2 edgeB
AddEdge A node1 node2 edgeA
AddEdge A node1 node2 edgeD
AddEdge A node1 node2 edgeC
AddEdge A node2 node1 edgeF
AddEdge A node2 node1 edgeG
AddEdge A node2 node1 edgeA
AddEdge A node2 node1 edgeB
AddEdge A node2 node1 edgeZ

# Print the nodes in the graph and checks its edges
ListNodes A
ListChildren A node1
ListChildren A node2