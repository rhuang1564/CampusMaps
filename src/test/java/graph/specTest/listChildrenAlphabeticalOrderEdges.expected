# Tests alphabetical ordering of listening of a node with multiple edges between same node

# Create a graph
created graph A

# Add a node
added node node1 to A
added node node2 to A

# Add edges
added edge edgeB from node1 to node2 in A
added edge edgeA from node1 to node2 in A
added edge edgeD from node1 to node2 in A
added edge edgeC from node1 to node2 in A
added edge edgeF from node2 to node1 in A
added edge edgeG from node2 to node1 in A
added edge edgeA from node2 to node1 in A
added edge edgeB from node2 to node1 in A
added edge edgeZ from node2 to node1 in A

# Print the nodes in the graph and checks its edges
A contains: node1 node2
the children of node1 in A are: node2(edgeA) node2(edgeB) node2(edgeC) node2(edgeD)
the children of node2 in A are: node1(edgeA) node1(edgeB) node1(edgeF) node1(edgeG) node1(edgeZ)